### PR TITLE
Cloud changelog - Nov 24

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by cwarnermm — APPROVED** [review]
Looks good! Thanks!

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -3,12 +3,46 @@
...
+ - Fixed an issue where push notifications did not clear from the lock screen or the notification center with Collapsed Reply Threads enabled.
+ - Fixed an issue where Direct Message notifications were missing the sender name with Collapsed Reply Threads enabled.
+ - Fixed an issue where keyboard shortcuts were not working with Global Threads.
+
+### Known Issues
+ - Member type is missing from autocomplete [MM-38989](https://mattermost.atlassian.net/browse/MM-38989).
+ - File upload might fail for SVG files [MM-38982](https://mattermost.atlassian.net/browse/MM-38982).
+ - ``CMD+/`` does not close the shortcuts modal [MM-38971](https://mattermost.atlassian.net/browse/MM-38971).
+ - ``Ctrl/Cmd+Shift+A`` shortcut does not open **Account Settings** [MM-38236](https://mattermost.atlassian.net/browse/MM-38236).
+ - Close button on the Invite People page is incorrectly themed [MM-37852](https://mattermost.atlassian.net/browse/MM-37852).
```
Comment (marianunez) [inline source/install/cloud-changelog.md:42]: Given that the invite people screen was redesigned, this bug no longer applies. Not sure if you want to add something regarding the new invite screen https://mattermost.atlassian.net/browse/MM-38799

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -3,12 +3,46 @@
...
+ - Known issues related to the new Collapsed Reply Threads (Beta) are [listed here](https://docs.mattermost.com/messaging/organizing-conversations.html#known-issues).
+ - **Cloud > "Tips & Next Steps"** should not show an "Explore channels" section for guest users.
```
Comment (marianunez) [inline source/install/cloud-changelog.md:44]: This one is also not an issue any longer as Tips & Next Steps have been completly removed https://mattermost.atlassian.net/browse/MM-39057

**Review by marianunez — APPROVED** [review]
LGTM, minor comments on known issues

**Review by agnivade — APPROVED** [review]

**Review by gabrieljackson — APPROVED** [review]

**Review by esethna — APPROVED** [review]

**Pull request merged**